tutor/index.coffee

```diff
@@ -31,6 +31,9 @@ window._STORES =
   NOTIFICATIONS: require './src/flux/notifications'
   TOC: require './src/flux/toc'
 
+# In dev builds this enables hot-reloading,
+# in production it simply renders the root app
+{AppContainer} = require 'react-hot-loader'
```

Comment:
hmm...interesting choice from `react-hot-loader to require `react` code for what used to be a setting...should we add this to `exercises` and `coach` and/or make a `shared` helper function that will handle this?